manuscript/090_How_to_start.md

```diff
@@ -284,14 +286,16 @@ ShouldAllowAccessToReportingWhenAskedForEitherAdministratorOrAuditor()
 }
 ```
 
-Start by invoking a�method when you have one
---------------------------------------------
+We still need to give this Statement a name, but given what we already know this is an easy task.
```

Comment:
"Using what we learned by formulating the Statement, it was easy to give it a name."